module-network.tf

```diff
@@ -2,7 +2,7 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
 data "oci_core_vcn" "oke" {
-  count  = coalesce(var.vcn_id, "none") != "none" ? 1 : 0
+  count  = var.create_vcn ? 0 : 1
```

Comment:
This data source is only used here: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/module-network.tf#L14-L16

As it currently works, if `var.create_vcn == true` this gets ignored entirely, and if `var.create_vcn == false` and `var.vcn_id` is empty, then this module fails elsewhere already